src/smp.rs

```diff
@@ -18,6 +20,8 @@ use crate::vm_id::VmIdTracker;
 
 // The secondary CPU entry point, defined in start.S.
```

Comment:
move this comment to be over `_secondary_start` because it doesn't really apply to `_stack_*`

Reply:
Right.

Reply:
Done.